packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -214,15 +214,16 @@ export class EthereumAnchorValidator implements AnchorValidator {
       throw new Error(`The root CID ${anchorProof.root} is not in the transaction`)
     }
 
-    if (txResponse.blockNumber <= BLOCK_THRESHHOLDS[this._chainId]) {
+    // we use the anchor proof chain id instead of the this._chainId, as this._chainId is not set if the ceramic node is being used in readOnly mode
+    if (txResponse.blockNumber <= BLOCK_THRESHHOLDS[anchorProof.chainId]) {
```

Comment:
can we add an extra check so if the chainId is one we don't understand we throw a nice error message?

Reply:
that will happen right here: https://github.com/ceramicnetwork/js-ceramic/blob/765e89295ee94d745d52359022fbbe83fc77fc07/packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts#L249

Is this error message sufficient?

Reply:
ah yep, that works